BuildResidentialHPXML/measure.rb

```diff
@@ -1025,18 +1029,17 @@ def arguments(model)
     arg.setDefaultValue(HPXML::HVACTypeCentralAirConditioner)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_cooling_efficiency_seer', true)
-    arg.setDisplayName('Cooling System: Rated SEER')
-    arg.setUnits('SEER')
-    arg.setDescription("The rated efficiency value of the #{HPXML::HVACTypeCentralAirConditioner} cooling system.")
-    arg.setDefaultValue(13.0)
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('cooling_system_cooling_efficiency_type', cooling_efficiency_type_choices, true)
+    arg.setDisplayName('Cooling System: Efficiency Type')
+    arg.setDescription('The efficiency type of the cooling system.')
+    arg.setDefaultValue(HPXML::UnitsSEER)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_cooling_efficiency_eer', true)
-    arg.setDisplayName('Cooling System: Rated EER')
-    arg.setUnits('EER')
-    arg.setDescription("The rated efficiency value of the #{HPXML::HVACTypeRoomAirConditioner} cooling system.")
-    arg.setDefaultValue(8.5)
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_cooling_efficiency', true)
+    arg.setDisplayName('Cooling System: Efficiency')
+    arg.setUnits("#{HPXML::UnitsSEER} or #{HPXML::UnitsEER}")
+    arg.setDescription('The rated efficiency value of the cooling system.')
```

Comment:
Need to retain which system types use SEER and which use EER in the argument description.

Reply:
Actually, why do we need the `cooling_system_cooling_efficiency_type` argument at all if the efficiency type corresponds to the system type?

Reply:
I think it's safer. For a central air conditioner, if someone entered its EER instead of SEER, they would get an incorrect simulation result if we removed the argument and not even realize that the value was treated as a SEER. But as it stands now, if they entered EER instead of SEER, they would get an error that SEER is expected.